overlord/snapstate/snapstate_update_test.go

```diff
@@ -13796,10 +13796,10 @@ func (s *snapmgrTestSuite) TestUpdateStateConflictRemoved(c *C) {
 
 func (s *snapmgrTestSuite) TestUpdateWithComponentsBackToPrevRevision(c *C) {
```

Comment:
shouldn't we have kept both a version with and without components of this?

Reply:
I can add the old version back if you like, but this is really just additive, the functionality of the original test is still there.